translations/base-ind.yaml

```diff
@@ -142,7 +142,7 @@ mainMenu:
     play: Mulai Permainan
     continue: Lanjutkan Permainan
     newGame: Permainan Baru
-    changelog: Ganti Data Log
+    changelog: Catatan Perubahan 
```

Comment:
You added an unnecessary space after the translation. These are called `trailing spaces` and cause the build to produce error messages. Please remove it and check for them in the future